### ISSUE TITLE
Not installing properly

### ISSUE THREAD

**Issue by Redteamer0101** [vmware-labs/attack-surface-framework#19]
### Describe the bug

I tried install by given steps but after installation in browser it gives nginx 404 not found page. while installing it asked for mongo db steps so I selected local and username and password nothing else. Also one request please can you provide docker for this project it will be very help full for installation issues. Thank you so much.

### Reproduction steps

1. Install asf
2. once installation is done try accessing in browser
3. try 127.0.0.1:2021

### Expected behavior

It has to open up registration page.

**Comment by pavsorab** [comment]
@Redteamer0101 Thank you for your detailed feedback and for bringing this to our attention!

We will investigate this bug and will actively work towards providing a fix in the next update. And we absolutely appreciate your suggestion regarding providing a Docker container for ASF to ease the installation process.

**Comment by MorielHarush** [comment]
> @Redteamer0101 Thank you for your detailed feedback and for bringing this to our attention!
> 
> We will investigate this bug and will actively work towards providing a fix in the next update. And we absolutely appreciate your suggestion regarding providing a Docker container for ASF to ease the installation process.

Hey, it is still gives 404 error after installation.

**Comment by Redteamer0101** [comment]
@MorielHarush Yes same error is there

**Comment by MorielHarush** [comment]
> @MorielHarush Yes same error is there

@pavsorab - Any answer about that ? is there any fix upcoming ?

**Comment by Dezz27** [comment]
> @pavsorab - Any answer about that ? is there any fix upcoming ?

@pavsorab Hello, is there any fix?

**Comment by pavsorab** [comment]
@MorielHarush @Dezz27 Sorry for the delay in response. I have a pending PR that needs to be merged. I will reach out to concerned person and get back to you.

**Comment by Redteamer0101** [comment]
Hi Team, now it got installed but while accessing http://127.0.0.1:2021/ it gives error of Bad Request (400). I'm using kali uptodate. in Virtualbox.

Here is what it comes while running setup.sh

┌──(kali㉿kali)-[/opt/asf]
└─$ sudo ./setup.sh
Hit:2 http://downloads.metasploit.com/data/releases/metasploit-framework/apt lucid InRelease
Get:3 http://kali.download/kali kali-rolling InRelease [41.5 kB]
Ign:1 https://packages.cloud.google.com/apt kubernetes-xenial InRelease
Get:5 http://kali.download/kali kali-rolling/main amd64 Packages [19.8 MB]
Err:4 https://packages.cloud.google.com/apt kubernetes-xenial Release
  404  Not Found [IP: 2404:6800:4009:80c::200e 443]
Get:6 http://kali.download/kali kali-rolling/main amd64 Contents (deb) [46.5 MB]
Reading package lists... Done                                                                                                                               
W: http://downloads.metasploit.com/data/releases/metasploit-framework/apt/dists/lucid/InRelease: Key is stored in legacy trusted.gpg keyring (/etc/apt/trusted.gpg), see the DEPRECATION section in apt-key(8) for details.
E: The repository 'https://apt.kubernetes.io kubernetes-xenial Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
fatal: destination path '/home/nuclei-templates' already exists and is not an empty directory.
nohup: appending output to 'nohup.out'
Collecting asgiref (from -r requirements.txt (line 1))
  Obtaining dependency information for asgiref from https://files.pythonhosted.org/packages/39/e3/893e8757be2612e6c266d9bb58ad2e3651524b5b40cf56761e985a28b13e/asgiref-3.8.1-py3-none-any.whl.metadata
  Downloading asgiref-3.8.1-py3-none-any.whl.metadata (9.3 kB)
Collecting autopep8 (from -r requirements.txt (line 2))
  Obtaining dependency information for autopep8 from https://files.pythonhosted.org/packages/7b/09/87d32f364e09faebd126b2e52609182ce71ecc2ccf7e6daf8889704756b7/autopep8-2.1.0-py2.py3-none-any.whl.metadata
  Downloading autopep8-2.1.0-py2.py3-none-any.whl.metadata (16 kB)
Collecting django (from -r requirements.txt (line 3))
  Obtaining dependency information for django from https://files.pythonhosted.org/packages/1d/23/02f3795a71196019bcfec4c67890a6369e43b023474154fa0b2b7060346d/Django-5.0.6-py3-none-any.whl.metadata
  Downloading Django-5.0.6-py3-none-any.whl.metadata (4.1 kB)
Collecting pycodestyle (from -r requirements.txt (line 4))
  Obtaining dependency information for pycodestyle from https://files.pythonhosted.org/packages/b1/90/a998c550d0ddd07e38605bb5c455d00fcc177a800ff9cc3dafdcb3dd7b56/pycodestyle-2.11.1-py2.py3-none-any.whl.metadata
  Downloading pycodestyle-2.11.1-py2.py3-none-any.whl.metadata (4.5 kB)
Collecting pytz (from -r requirements.txt (line 5))
  Obtaining dependency information for pytz from https://files.pythonhosted.org/packages/9c/3d/a121f284241f08268b21359bd425f7d4825cffc5ac5cd0e1b3d82ffd2b10/pytz-2024.1-py2.py3-none-any.whl.metadata
  Downloading pytz-2024.1-py2.py3-none-any.whl.metadata (22 kB)
Collecting sqlparse (from -r requirements.txt (line 6))
  Obtaining dependency information for sqlparse from https://files.pythonhosted.org/packages/43/5d/a0fdd88fd486b39ae1fd1a75ff75b4e29a0df96c0304d462fd407b82efe0/sqlparse-0.5.0-py3-none-any.whl.metadata
  Downloading sqlparse-0.5.0-py3-none-any.whl.metadata (3.9 kB)
Collecting Unipath (from -r requirements.txt (line 7))
  Using cached Unipath-1.1-py3-none-any.whl
Collecting dj-database-url (from -r requirements.txt (line 8))
  Obtaining dependency information for dj-database-url from https://files.pythonhosted.org/packages/21/d2/b90366de4682c1472748b848292054e0960673f3c7d1a9d08fb227fad819/dj_database_url-2.1.0-py3-none-any.whl.metadata
  Using cached dj_database_url-2.1.0-py3-none-any.whl.metadata (11 kB)
Collecting python-decouple (from -r requirements.txt (line 9))
  Obtaining dependency information for python-decouple from https://files.pythonhosted.org/packages/a2/d4/9193206c4563ec771faf2ccf54815ca7918529fe81f6adb22ee6d0e06622/python_decouple-3.8-py3-none-any.whl.metadata
  Downloading python_decouple-3.8-py3-none-any.whl.metadata (14 kB)
Collecting gunicorn (from -r requirements.txt (line 10))
  Obtaining dependency information for gunicorn from https://files.pythonhosted.org/packages/29/97/6d610ae77b5633d24b69c2ff1ac3044e0e565ecbd1ec188f02c45073054c/gunicorn-22.0.0-py3-none-any.whl.metadata
  Downloading gunicorn-22.0.0-py3-none-any.whl.metadata (4.4 kB)
Collecting whitenoise (from -r requirements.txt (line 11))
  Obtaining dependency information for whitenoise from https://files.pythonhosted.org/packages/67/16/bb488ac8230f1bce94943b6654f2aad566d18aae575c8b6d8a99c78c489e/whitenoise-6.6.0-py3-none-any.whl.metadata
  Downloading whitenoise-6.6.0-py3-none-any.whl.metadata (3.7 kB)
Collecting sqlalchemy (from -r requirements.txt (line 12))
  Obtaining dependency information for sqlalchemy from https://files.pythonhosted.org/packages/4d/21/87bcad723070f7cd5f9d45fb05557596aa1d23d19eef078b13edc9e31813/SQLAlchemy-2.0.30-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata
  Downloading SQLAlchemy-2.0.30-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (9.6 kB)
Collecting webscreenshot (from -r requirements.txt (line 13))
  Using cached webscreenshot-2.94-py3-none-any.whl
Collecting netaddr (from -r requirements.txt (line 14))
  Obtaining dependency information for netaddr from https://files.pythonhosted.org/packages/d0/e4/a57d2ad0c1381d6304c7eb3aed7c1a415c5b443e71d7e5c88378ac60d1ef/netaddr-1.2.1-py3-none-any.whl.metadata
  Downloading netaddr-1.2.1-py3-none-any.whl.metadata (5.0 kB)
Collecting pymetasploit3 (from -r requirements.txt (line 15))
  Downloading pymetasploit3-1.0.6.tar.gz (24 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Collecting psutil (from -r requirements.txt (line 16))
  Obtaining dependency information for psutil from https://files.pythonhosted.org/packages/c5/4f/0e22aaa246f96d6ac87fe5ebb9c5a693fbe8877f537a1022527c47ca43c5/psutil-5.9.8-cp36-abi3-manylinux_2_12_x86_64.manylinux2010_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata
  Downloading psutil-5.9.8-cp36-abi3-manylinux_2_12_x86_64.manylinux2010_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (21 kB)
Collecting django-allauth (from -r requirements.txt (line 17))
  Downloading django-allauth-0.63.1.tar.gz (1.1 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.1/1.1 MB 692.5 kB/s eta 0:00:00
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Installing backend dependencies ... done
  Preparing metadata (pyproject.toml) ... done
Collecting django-environ (from -r requirements.txt (line 18))
  Obtaining dependency information for django-environ from https://files.pythonhosted.org/packages/c4/f1/468b49cccba3b42dda571063a14c668bb0b53a1d5712426d18e36663bd53/django_environ-0.11.2-py2.py3-none-any.whl.metadata
  Using cached django_environ-0.11.2-py2.py3-none-any.whl.metadata (11 kB)
Collecting djangosaml2 (from -r requirements.txt (line 19))
  Obtaining dependency information for djangosaml2 from https://files.pythonhosted.org/packages/50/22/2f54ad74c28f6f4be6d8521b37cecbaa4820e0bf19285ce125d93ba20155/djangosaml2-1.9.3-py2.py3-none-any.whl.metadata
  Downloading djangosaml2-1.9.3-py2.py3-none-any.whl.metadata (3.2 kB)
Collecting pymongo (from -r requirements.txt (line 20))
  Obtaining dependency information for pymongo from https://files.pythonhosted.org/packages/b1/9a/d7cca333e6fcecd4300aa957a4d885424395590b7da5a81d638a92102416/pymongo-4.7.2-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata
  Using cached pymongo-4.7.2-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (22 kB)
Collecting jira (from -r requirements.txt (line 21))
  Obtaining dependency information for jira from https://files.pythonhosted.org/packages/4f/52/bb617020064261ba31cc965e932943458b7facfd9691ad7f76a2b631f44f/jira-3.8.0-py3-none-any.whl.metadata
  Downloading jira-3.8.0-py3-none-any.whl.metadata (9.1 kB)
Collecting typing-extensions>=3.10.0.0 (from dj-database-url->-r requirements.txt (line 8))
  Obtaining dependency information for typing-extensions>=3.10.0.0 from https://files.pythonhosted.org/packages/01/f3/936e209267d6ef7510322191003885de524fc48d1b43269810cd589ceaf5/typing_extensions-4.11.0-py3-none-any.whl.metadata
  Downloading typing_extensions-4.11.0-py3-none-any.whl.metadata (3.0 kB)
Collecting packaging (from gunicorn->-r requirements.txt (line 10))
  Obtaining dependency information for packaging from https://files.pythonhosted.org/packages/49/df/1fceb2f8900f8639e278b056416d49134fb8d84c5942ffaa01ad34782422/packaging-24.0-py3-none-any.whl.metadata
  Downloading packaging-24.0-py3-none-any.whl.metadata (3.2 kB)
Collecting greenlet!=0.4.17 (from sqlalchemy->-r requirements.txt (line 12))
  Obtaining dependency information for greenlet!=0.4.17 from https://files.pythonhosted.org/packages/c6/1f/12d5a6cc26e8b483c2e7975f9c22e088ac735c0d8dcb8a8f72d31a4e5f04/greenlet-3.0.3-cp311-cp311-manylinux_2_24_x86_64.manylinux_2_28_x86_64.whl.metadata
  Downloading greenlet-3.0.3-cp311-cp311-manylinux_2_24_x86_64.manylinux_2_28_x86_64.whl.metadata (3.8 kB)
Collecting argparse (from webscreenshot->-r requirements.txt (line 13))
  Obtaining dependency information for argparse from https://files.pythonhosted.org/packages/f2/94/3af39d34be01a24a6e65433d19e107099374224905f1e0cc6bbe1fd22a2f/argparse-1.4.0-py2.py3-none-any.whl.metadata
  Using cached argparse-1.4.0-py2.py3-none-any.whl.metadata (2.8 kB)
Collecting future (from webscreenshot->-r requirements.txt (line 13))
  Obtaining dependency information for future from https://files.pythonhosted.org/packages/da/71/ae30dadffc90b9006d77af76b393cb9dfbfc9629f339fc1574a1c52e6806/future-1.0.0-py3-none-any.whl.metadata
  Downloading future-1.0.0-py3-none-any.whl.metadata (4.0 kB)
Collecting msgpack (from pymetasploit3->-r requirements.txt (line 15))
  Obtaining dependency information for msgpack from https://files.pythonhosted.org/packages/f6/f0/a7bdb48223cd21b9abed814b08fca8fe6a40931e70ec97c24d2f15d68ef3/msgpack-1.0.8-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata
  Downloading msgpack-1.0.8-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (9.1 kB)
Collecting requests (from pymetasploit3->-r requirements.txt (line 15))
  Obtaining dependency information for requests from https://files.pythonhosted.org/packages/c3/20/748e38b466e0819491f0ce6e90ebe4184966ee304fe483e2c414b0f4ef07/requests-2.32.2-py3-none-any.whl.metadata
  Downloading requests-2.32.2-py3-none-any.whl.metadata (4.6 kB)
Collecting retry (from pymetasploit3->-r requirements.txt (line 15))
  Obtaining dependency information for retry from https://files.pythonhosted.org/packages/4b/0d/53aea75710af4528a25ed6837d71d117602b01946b307a3912cb3cfcbcba/retry-0.9.2-py2.py3-none-any.whl.metadata
  Downloading retry-0.9.2-py2.py3-none-any.whl.metadata (5.8 kB)
Collecting defusedxml>=0.4.1 (from djangosaml2->-r requirements.txt (line 19))
  Obtaining dependency information for defusedxml>=0.4.1 from https://files.pythonhosted.org/packages/07/6c/aa3f2f849e01cb6a001cd8554a88d4c77c5c1a31c95bdf1cf9301e6d9ef4/defusedxml-0.7.1-py2.py3-none-any.whl.metadata
  Downloading defusedxml-0.7.1-py2.py3-none-any.whl.metadata (32 kB)
Collecting pysaml2>=6.5.1 (from djangosaml2->-r requirements.txt (line 19))
  Obtaining dependency information for pysaml2>=6.5.1 from https://files.pythonhosted.org/packages/fe/d1/92d84ae0e80e829e84785c6e4e425ff6d447116289f0ecf2af068f771a73/pysaml2-7.5.0-py3-none-any.whl.metadata
  Downloading pysaml2-7.5.0-py3-none-any.whl.metadata (7.1 kB)
Collecting dnspython<3.0.0,>=1.16.0 (from pymongo->-r requirements.txt (line 20))
  Obtaining dependency information for dnspython<3.0.0,>=1.16.0 from https://files.pythonhosted.org/packages/87/a1/8c5287991ddb8d3e4662f71356d9656d91ab3a36618c3dd11b280df0d255/dnspython-2.6.1-py3-none-any.whl.metadata
  Downloading dnspython-2.6.1-py3-none-any.whl.metadata (5.8 kB)
Collecting Pillow>=2.1.0 (from jira->-r requirements.txt (line 21))
  Obtaining dependency information for Pillow>=2.1.0 from https://files.pythonhosted.org/packages/81/ff/ad3c942d865f9e45ce84eeb31795e6d4d94e1f1eea51026d5154028510d7/pillow-10.3.0-cp311-cp311-manylinux_2_28_x86_64.whl.metadata
  Downloading pillow-10.3.0-cp311-cp311-manylinux_2_28_x86_64.whl.metadata (9.2 kB)
Collecting requests-oauthlib>=1.1.0 (from jira->-r requirements.txt (line 21))
  Obtaining dependency information for requests-oauthlib>=1.1.0 from https://files.pythonhosted.org/packages/3b/5d/63d4ae3b9daea098d5d6f5da83984853c1bbacd5dc826764b249fe119d24/requests_oauthlib-2.0.0-py2.py3-none-any.whl.metadata
  Downloading requests_oauthlib-2.0.0-py2.py3-none-any.whl.metadata (11 kB)
Collecting requests-toolbelt (from jira->-r requirements.txt (line 21))
  Obtaining dependency information for requests-toolbelt from https://files.pythonhosted.org/packages/3f/51/d4db610ef29373b879047326cbf6fa98b6c1969d6f6dc423279de2b1be2c/requests_toolbelt-1.0.0-py2.py3-none-any.whl.metadata
  Downloading requests_toolbelt-1.0.0-py2.py3-none-any.whl.metadata (14 kB)
Collecting cryptography>=3.1 (from pysaml2>=6.5.1->djangosaml2->-r requirements.txt (line 19))
  Obtaining dependency information for cryptography>=3.1 from https://files.pythonhosted.org/packages/7b/4e/fa4896744259ee8602464ed2c7330b736cc4dd3fd92f63cd56828bf36707/cryptography-42.0.7-cp39-abi3-manylinux_2_28_x86_64.whl.metadata
  Downloading cryptography-42.0.7-cp39-abi3-manylinux_2_28_x86_64.whl.metadata (5.3 kB)
Collecting pyopenssl (from pysaml2>=6.5.1->djangosaml2->-r requirements.txt (line 19))
  Obtaining dependency information for pyopenssl from https://files.pythonhosted.org/packages/54/a7/2104f674a5a6845b04c8ff01659becc6b8978ca410b82b94287e0b1e018b/pyOpenSSL-24.1.0-py3-none-any.whl.metadata
  Downloading pyOpenSSL-24.1.0-py3-none-any.whl.metadata (12 kB)
Collecting python-dateutil (from pysaml2>=6.5.1->djangosaml2->-r requirements.txt (line 19))
  Obtaining dependency information for python-dateutil from https://files.pythonhosted.org/packages/ec/57/56b9bcc3c9c6a792fcbaf139543cee77261f3651ca9da0c93f5c1221264b/python_dateutil-2.9.0.post0-py2.py3-none-any.whl.metadata
  Downloading python_dateutil-2.9.0.post0-py2.py3-none-any.whl.metadata (8.4 kB)
Collecting xmlschema<3,>=2 (from pysaml2>=6.5.1->djangosaml2->-r requirements.txt (line 19))
  Obtaining dependency information for xmlschema<3,>=2 from https://files.pythonhosted.org/packages/5e/2a/c2bc97fd20efe65cfcfc21666d1b0213969133d37ea093761d264d9ed9f8/xmlschema-2.5.1-py3-none-any.whl.metadata
  Downloading xmlschema-2.5.1-py3-none-any.whl.metadata (8.3 kB)
Collecting charset-normalizer<4,>=2 (from requests->pymetasploit3->-r requirements.txt (line 15))
  Obtaining dependency information for charset-normalizer<4,>=2 from https://files.pythonhosted.org/packages/40/26/f35951c45070edc957ba40a5b1db3cf60a9dbb1b350c2d5bef03e01e61de/charset_normalizer-3.3.2-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata
  Downloading charset_normalizer-3.3.2-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (33 kB)
Collecting idna<4,>=2.5 (from requests->pymetasploit3->-r requirements.txt (line 15))
  Obtaining dependency information for idna<4,>=2.5 from https://files.pythonhosted.org/packages/e5/3e/741d8c82801c347547f8a2a06aa57dbb1992be9e948df2ea0eda2c8b79e8/idna-3.7-py3-none-any.whl.metadata
  Downloading idna-3.7-py3-none-any.whl.metadata (9.9 kB)
Collecting urllib3<3,>=1.21.1 (from requests->pymetasploit3->-r requirements.txt (line 15))
  Obtaining dependency information for urllib3<3,>=1.21.1 from https://files.pythonhosted.org/packages/a2/73/a68704750a7679d0b6d3ad7aa8d4da8e14e151ae82e6fee774e6e0d05ec8/urllib3-2.2.1-py3-none-any.whl.metadata
  Downloading urllib3-2.2.1-py3-none-any.whl.metadata (6.4 kB)
Collecting certifi>=2017.4.17 (from requests->pymetasploit3->-r requirements.txt (line 15))
  Obtaining dependency information for certifi>=2017.4.17 from https://files.pythonhosted.org/packages/ba/06/a07f096c664aeb9f01624f858c3add0a4e913d6c96257acb4fce61e7de14/certifi-2024.2.2-py3-none-any.whl.metadata
  Downloading certifi-2024.2.2-py3-none-any.whl.metadata (2.2 kB)
Collecting oauthlib>=3.0.0 (from requests-oauthlib>=1.1.0->jira->-r requirements.txt (line 21))
  Obtaining dependency information for oauthlib>=3.0.0 from https://files.pythonhosted.org/packages/7e/80/cab10959dc1faead58dc8384a781dfbf93cb4d33d50988f7a69f1b7c9bbe/oauthlib-3.2.2-py3-none-any.whl.metadata
  Downloading oauthlib-3.2.2-py3-none-any.whl.metadata (7.5 kB)
Collecting decorator>=3.4.2 (from retry->pymetasploit3->-r requirements.txt (line 15))
  Obtaining dependency information for decorator>=3.4.2 from https://files.pythonhosted.org/packages/d5/50/83c593b07763e1161326b3b8c6686f0f4b0f24d5526546bee538c89837d6/decorator-5.1.1-py3-none-any.whl.metadata
  Downloading decorator-5.1.1-py3-none-any.whl.metadata (4.0 kB)
Collecting py<2.0.0,>=1.4.26 (from retry->pymetasploit3->-r requirements.txt (line 15))
  Obtaining dependency information for py<2.0.0,>=1.4.26 from https://files.pythonhosted.org/packages/f6/f0/10642828a8dfb741e5f3fbaac830550a518a775c7fff6f04a007259b0548/py-1.11.0-py2.py3-none-any.whl.metadata
  Downloading py-1.11.0-py2.py3-none-any.whl.metadata (2.8 kB)
Collecting cffi>=1.12 (from cryptography>=3.1->pysaml2>=6.5.1->djangosaml2->-r requirements.txt (line 19))
  Obtaining dependency information for cffi>=1.12 from https://files.pythonhosted.org/packages/9b/89/a31c81e36bbb793581d8bba4406a8aac4ba84b2559301c44eef81f4cf5df/cffi-1.16.0-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata
  Using cached cffi-1.16.0-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (1.5 kB)
Collecting elementpath<5.0.0,>=4.1.5 (from xmlschema<3,>=2->pysaml2>=6.5.1->djangosaml2->-r requirements.txt (line 19))
  Obtaining dependency information for elementpath<5.0.0,>=4.1.5 from https://files.pythonhosted.org/packages/36/8a/7ba4eec818455d4ee941e8a50b2ffc98a2d2627156c9e79506411e0970e3/elementpath-4.4.0-py3-none-any.whl.metadata
  Downloading elementpath-4.4.0-py3-none-any.whl.metadata (6.7 kB)
Collecting six>=1.5 (from python-dateutil->pysaml2>=6.5.1->djangosaml2->-r requirements.txt (line 19))
  Obtaining dependency information for six>=1.5 from https://files.pythonhosted.org/packages/d9/5a/e7c31adbe875f2abbb91bd84cf2dc52d792b5a01506781dbcf25c91daf11/six-1.16.0-py2.py3-none-any.whl.metadata
  Downloading six-1.16.0-py2.py3-none-any.whl.metadata (1.8 kB)
Collecting pycparser (from cffi>=1.12->cryptography>=3.1->pysaml2>=6.5.1->djangosaml2->-r requirements.txt (line 19))
  Obtaining dependency information for pycparser from https://files.pythonhosted.org/packages/13/a3/a812df4e2dd5696d1f351d58b8fe16a405b234ad2886a0dab9183fb78109/pycparser-2.22-py3-none-any.whl.metadata
  Downloading pycparser-2.22-py3-none-any.whl.metadata (943 bytes)
Downloading asgiref-3.8.1-py3-none-any.whl (23 kB)
Downloading autopep8-2.1.0-py2.py3-none-any.whl (44 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 45.0/45.0 kB 1.4 MB/s eta 0:00:00
Downloading Django-5.0.6-py3-none-any.whl (8.2 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 8.2/8.2 MB 1.5 MB/s eta 0:00:00
Downloading pycodestyle-2.11.1-py2.py3-none-any.whl (31 kB)
Downloading pytz-2024.1-py2.py3-none-any.whl (505 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 505.5/505.5 kB 1.1 MB/s eta 0:00:00
Downloading sqlparse-0.5.0-py3-none-any.whl (43 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 44.0/44.0 kB 1.3 MB/s eta 0:00:00
Using cached dj_database_url-2.1.0-py3-none-any.whl (7.7 kB)
Using cached python_decouple-3.8-py3-none-any.whl (9.9 kB)
Downloading gunicorn-22.0.0-py3-none-any.whl (84 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 84.4/84.4 kB 2.6 MB/s eta 0:00:00
Downloading whitenoise-6.6.0-py3-none-any.whl (19 kB)
Downloading SQLAlchemy-2.0.30-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (3.2 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.2/3.2 MB 1.3 MB/s eta 0:00:00
Downloading netaddr-1.2.1-py3-none-any.whl (2.3 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.3/2.3 MB 1.3 MB/s eta 0:00:00
Downloading psutil-5.9.8-cp36-abi3-manylinux_2_12_x86_64.manylinux2010_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl (288 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 288.2/288.2 kB 1.5 MB/s eta 0:00:00
Using cached django_environ-0.11.2-py2.py3-none-any.whl (19 kB)
Downloading djangosaml2-1.9.3-py2.py3-none-any.whl (66 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 67.0/67.0 kB 1.3 MB/s eta 0:00:00
Using cached pymongo-4.7.2-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (672 kB)
Downloading jira-3.8.0-py3-none-any.whl (77 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 77.5/77.5 kB 2.5 MB/s eta 0:00:00
Using cached defusedxml-0.7.1-py2.py3-none-any.whl (25 kB)
Downloading dnspython-2.6.1-py3-none-any.whl (307 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 307.7/307.7 kB 1.6 MB/s eta 0:00:00
Downloading greenlet-3.0.3-cp311-cp311-manylinux_2_24_x86_64.manylinux_2_28_x86_64.whl (620 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 620.0/620.0 kB 1.4 MB/s eta 0:00:00
Downloading pillow-10.3.0-cp311-cp311-manylinux_2_28_x86_64.whl (4.5 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 4.5/4.5 MB 1.4 MB/s eta 0:00:00
Downloading pysaml2-7.5.0-py3-none-any.whl (419 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 419.3/419.3 kB 1.5 MB/s eta 0:00:00
Downloading requests-2.32.2-py3-none-any.whl (63 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 63.9/63.9 kB 1.0 MB/s eta 0:00:00
Downloading requests_oauthlib-2.0.0-py2.py3-none-any.whl (24 kB)
Downloading typing_extensions-4.11.0-py3-none-any.whl (34 kB)
Using cached argparse-1.4.0-py2.py3-none-any.whl (23 kB)
Downloading future-1.0.0-py3-none-any.whl (491 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 491.3/491.3 kB 646.0 kB/s eta 0:00:00
Downloading msgpack-1.0.8-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (409 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 409.3/409.3 kB 1.5 MB/s eta 0:00:00
Downloading packaging-24.0-py3-none-any.whl (53 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 53.5/53.5 kB 1.0 MB/s eta 0:00:00
Using cached requests_toolbelt-1.0.0-py2.py3-none-any.whl (54 kB)
Using cached retry-0.9.2-py2.py3-none-any.whl (8.0 kB)
Downloading certifi-2024.2.2-py3-none-any.whl (163 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 163.8/163.8 kB 1.1 MB/s eta 0:00:00
Downloading charset_normalizer-3.3.2-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (140 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 140.3/140.3 kB 1.3 MB/s eta 0:00:00
Downloading cryptography-42.0.7-cp39-abi3-manylinux_2_28_x86_64.whl (3.8 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.8/3.8 MB 1.5 MB/s eta 0:00:00
Using cached decorator-5.1.1-py3-none-any.whl (9.1 kB)
Downloading idna-3.7-py3-none-any.whl (66 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 66.8/66.8 kB 1.4 MB/s eta 0:00:00
Using cached oauthlib-3.2.2-py3-none-any.whl (151 kB)
Using cached py-1.11.0-py2.py3-none-any.whl (98 kB)
Downloading urllib3-2.2.1-py3-none-any.whl (121 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 121.1/121.1 kB 1.2 MB/s eta 0:00:00
Downloading xmlschema-2.5.1-py3-none-any.whl (395 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 395.1/395.1 kB 1.3 MB/s eta 0:00:00
Downloading pyOpenSSL-24.1.0-py3-none-any.whl (56 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 56.9/56.9 kB 2.2 MB/s eta 0:00:00
Downloading python_dateutil-2.9.0.post0-py2.py3-none-any.whl (229 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 229.9/229.9 kB 1.9 MB/s eta 0:00:00
Using cached cffi-1.16.0-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (464 kB)
Downloading elementpath-4.4.0-py3-none-any.whl (219 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 219.8/219.8 kB 1.3 MB/s eta 0:00:00
Using cached six-1.16.0-py2.py3-none-any.whl (11 kB)
Downloading pycparser-2.22-py3-none-any.whl (117 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 117.6/117.6 kB 1.2 MB/s eta 0:00:00
Building wheels for collected packages: pymetasploit3, django-allauth
  Building wheel for pymetasploit3 (pyproject.toml) ... done
  Created wheel for pymetasploit3: filename=pymetasploit3-1.0.6-py3-none-any.whl size=22808 sha256=7db3d194e509734795cff317c5c163fe70a17c8ea6c02d2c9cc608f048c96a11
  Stored in directory: /root/.cache/pip/wheels/bc/24/f6/46fe3bac906bc76e216129874d1e6606e40889016b4d2d14cc
  Building wheel for django-allauth (pyproject.toml) ... done
  Created wheel for django-allauth: filename=django_allauth-0.63.1-py3-none-any.whl size=1416213 sha256=488ef24b8ec10556ddddf9d2a200f58c655c05f876bb394be3a13963bfa452aa
  Stored in directory: /root/.cache/pip/wheels/fb/89/ed/c13ac5ff404f4992cb1fb986bea9533236d830520ae8463f74
Successfully built pymetasploit3 django-allauth
Installing collected packages: Unipath, pytz, python-decouple, argparse, whitenoise, urllib3, typing-extensions, sqlparse, six, pycparser, pycodestyle, py, psutil, Pillow, packaging, oauthlib, netaddr, msgpack, idna, greenlet, future, elementpath, dnspython, django-environ, defusedxml, decorator, charset-normalizer, certifi, asgiref, xmlschema, webscreenshot, sqlalchemy, retry, requests, python-dateutil, pymongo, gunicorn, django, cffi, autopep8, requests-toolbelt, requests-oauthlib, pymetasploit3, django-allauth, dj-database-url, cryptography, pyopenssl, jira, pysaml2, djangosaml2
Successfully installed Pillow-10.3.0 Unipath-1.1 argparse-1.4.0 asgiref-3.8.1 autopep8-2.1.0 certifi-2024.2.2 cffi-1.16.0 charset-normalizer-3.3.2 cryptography-42.0.7 decorator-5.1.1 defusedxml-0.7.1 dj-database-url-2.1.0 django-5.0.6 django-allauth-0.63.1 django-environ-0.11.2 djangosaml2-1.9.3 dnspython-2.6.1 elementpath-4.4.0 future-1.0.0 greenlet-3.0.3 gunicorn-22.0.0 idna-3.7 jira-3.8.0 msgpack-1.0.8 netaddr-1.2.1 oauthlib-3.2.2 packaging-24.0 psutil-5.9.8 py-1.11.0 pycodestyle-2.11.1 pycparser-2.22 pymetasploit3-1.0.6 pymongo-4.7.2 pyopenssl-24.1.0 pysaml2-7.5.0 python-dateutil-2.9.0.post0 python-decouple-3.8 pytz-2024.1 requests-2.32.2 requests-oauthlib-2.0.0 requests-toolbelt-1.0.0 retry-0.9.2 six-1.16.0 sqlalchemy-2.0.30 sqlparse-0.5.0 typing-extensions-4.11.0 urllib3-2.2.1 webscreenshot-2.94 whitenoise-6.6.0 xmlschema-2.5.1
django-csp could not be found, not updating Content-Security-Policy. Please make sure CSP is configured. This can be done by your reverse proxy, django-csp or a custom CSP handler via SAML_CSP_HANDLER. See https://djangosaml2.readthedocs.io/contents/security.html#content-security-policy for more information. This warning can be disabled by setting `SAML_CSP_HANDLER=''` in your settings.
System check identified some issues:

WARNINGS:
app.vdInServices: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
        HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.                                                                                                                                           
app.vdInTarget: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
        HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.                                                                                                                                           
app.vdJob: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
        HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.                                                                                                                                           
app.vdNucleiResult: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
        HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.                                                                                                                                           
app.vdRegExp: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
        HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.                                                                                                                                           
app.vdResult: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
        HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.                                                                                                                                           
app.vdServices: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
        HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.                                                                                                                                           
app.vdTarget: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
        HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.                                                                                                                                           
Migrations for 'app':
  app/migrations/0001_initial.py
    - Create model vdInServices
    - Create model vdInTarget
    - Create model vdJob
    - Create model vdRegExp
    - Create model vdResult
    - Create model vdServices
    - Create model vdTarget
    - Create model vdNucleiResult
django-csp could not be found, not updating Content-Security-Policy. Please make sure CSP is configured. This can be done by your reverse proxy, django-csp or a custom CSP handler via SAML_CSP_HANDLER. See https://djangosaml2.readthedocs.io/contents/security.html#content-security-policy for more information. This warning can be disabled by setting `SAML_CSP_HANDLER=''` in your settings.
System check identified some issues:

WARNINGS:
app.vdInServices: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
        HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.                                                                                                                                           
app.vdInTarget: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
        HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.                                                                                                                                           
app.vdJob: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
        HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.                                                                                                                                           
app.vdNucleiResult: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
        HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.                                                                                                                                           
app.vdRegExp: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
        HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.                                                                                                                                           
app.vdResult: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
        HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.                                                                                                                                           
app.vdServices: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
        HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.                                                                                                                                           
app.vdTarget: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
        HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.                                                                                                                                           
Operations to perform:
  Apply all migrations: account, admin, app, auth, contenttypes, sessions, socialaccount
Running migrations:
  Applying contenttypes.0001_initial... OK
  Applying auth.0001_initial... OK
  Applying account.0001_initial... OK
  Applying account.0002_email_max_length... OK
  Applying account.0003_alter_emailaddress_create_unique_verified_email... OK
  Applying account.0004_alter_emailaddress_drop_unique_email... OK
  Applying account.0005_emailaddress_idx_upper_email... OK
  Applying account.0006_emailaddress_lower... OK
  Applying account.0007_emailaddress_idx_email... OK
  Applying account.0008_emailaddress_unique_primary_email_fixup... OK
  Applying account.0009_emailaddress_unique_primary_email... OK
  Applying admin.0001_initial... OK
  Applying admin.0002_logentry_remove_auto_add... OK
  Applying admin.0003_logentry_add_action_flag_choices... OK
  Applying app.0001_initial... OK
  Applying contenttypes.0002_remove_content_type_name... OK
  Applying auth.0002_alter_permission_name_max_length... OK
  Applying auth.0003_alter_user_email_max_length... OK
  Applying auth.0004_alter_user_username_opts... OK
  Applying auth.0005_alter_user_last_login_null... OK
  Applying auth.0006_require_contenttypes_0002... OK
  Applying auth.0007_alter_validators_add_error_messages... OK
  Applying auth.0008_alter_user_username_max_length... OK
  Applying auth.0009_alter_user_last_name_max_length... OK
  Applying auth.0010_alter_group_name_max_length... OK
  Applying auth.0011_update_proxy_permissions... OK
  Applying auth.0012_alter_user_first_name_max_length... OK
  Applying sessions.0001_initial... OK
  Applying socialaccount.0001_initial... OK
  Applying socialaccount.0002_token_max_lengths... OK
  Applying socialaccount.0003_extra_data_default_dict... OK
  Applying socialaccount.0004_app_provider_id_settings... OK
  Applying socialaccount.0005_socialtoken_nullable_app... OK
  Applying socialaccount.0006_alter_socialaccount_extra_data... OK
django-csp could not be found, not updating Content-Security-Policy. Please make sure CSP is configured. This can be done by your reverse proxy, django-csp or a custom CSP handler via SAML_CSP_HANDLER. See https://djangosaml2.readthedocs.io/contents/security.html#content-security-policy for more information. This warning can be disabled by setting `SAML_CSP_HANDLER=''` in your settings.
System check identified some issues:

WARNINGS:
app.vdInServices: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
        HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.                                                                                                                                           
app.vdInTarget: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
        HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.                                                                                                                                           
app.vdJob: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
        HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.                                                                                                                                           
app.vdNucleiResult: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
        HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.                                                                                                                                           
app.vdRegExp: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
        HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.                                                                                                                                           
app.vdResult: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
        HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.                                                                                                                                           
app.vdServices: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
        HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.                                                                                                                                           
app.vdTarget: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
        HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.                                                                                                                                           
Username (leave blank to use 'root'): root
Email address: root@mail.com
Password: 
Password (again): 
The password is too similar to the username.
This password is too short. It must contain at least 8 characters.                                                                                           
This password is too common.                                                                                                                                 
Bypass password validation and create user anyway? [y/N]: y                                                                                                  
Superuser created successfully.
Would you like to use kubernetes cluster (y/n)?
n
'asf.service' -> '/etc/systemd/system/asf.service'
'cleanuptrash.service' -> '/etc/systemd/system/cleanuptrash.service'
'cleanuptrash.timer' -> '/etc/systemd/system/cleanuptrash.timer'
'vdamass.asfui' -> '/etc/systemd/system/vdamass.asfui'
'vdamass.service' -> '/etc/systemd/system/vdamass.service'
'vdamass.timer' -> '/etc/systemd/system/vdamass.timer'
'vdexnmap.asfui' -> '/etc/systemd/system/vdexnmap.asfui'
'vdexnmap.service' -> '/etc/systemd/system/vdexnmap.service'
'vdexnmap.timer' -> '/etc/systemd/system/vdexnmap.timer'
Would you like to create a local MongoDB container for storing issues (y/n/d)?
y
This will run a local MongoDB container
Username for MongoDB:
root
MONGO_USER=root
Password:
MONGO_USER=root
docker: Error response from daemon: Conflict. The container name "/local-mongo" is already in use by container "1874bcf91d7a0aaa61cf973462838a83d50e513ef4edb5abd514b626621c67c0". You have to remove (or rename) that container to be able to reuse that name.
See 'docker run --help'.
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following packages were automatically installed and are no longer required:
  binutils-mingw-w64-i686 binutils-mingw-w64-x86-64 gcc-12-base gcc-mingw-w64-base gcc-mingw-w64-i686-win32 gcc-mingw-w64-i686-win32-runtime
  gcc-mingw-w64-x86-64-win32 gcc-mingw-w64-x86-64-win32-runtime gccgo-13 golang-1.20 golang-1.20-doc golang-1.20-go golang-1.20-src libaio1 libarmadillo11
  libcodec2-1.1 libgcc-12-dev libgo-13-dev libgo22 libgupnp-igd-1.0-4 libjim0.81 libnfs13 libobjc-12-dev libstdc++-12-dev linux-image-6.3.0-kali1-amd64
  mingw-w64-common mingw-w64-i686-dev mingw-w64-x86-64-dev oracle-instantclient-basic python3-jdcal
Use 'sudo apt autoremove' to remove them.
The following additional packages will be installed:
  libssl-dev libssl3t64 nginx-common openssl
Suggested packages:
  libssl-doc fcgiwrap nginx-doc
The following packages will be REMOVED:
  libssl3
The following NEW packages will be installed:
  libssl3t64
The following packages will be upgraded:
  libssl-dev nginx nginx-common openssl
4 upgraded, 1 newly installed, 1 to remove and 1998 not upgraded.
Need to get 2,699 kB/6,942 kB of archives.
After this operation, 3,457 kB of additional disk space will be used.
Get:1 http://kali.download/kali kali-rolling/main amd64 libssl-dev amd64 3.2.1-3 [2,699 kB]                                                                 
Fetched 2,699 kB in 13s (208 kB/s)                                                                                                                          
Preconfiguring packages ...
(Reading database ... 471396 files and directories currently installed.)
Preparing to unpack .../libssl-dev_3.2.1-3_amd64.deb ...
Unpacking libssl-dev:amd64 (3.2.1-3) over (3.0.10-1) ...
Preparing to unpack .../openssl_3.2.1-3_amd64.deb ...
Unpacking openssl (3.2.1-3) over (3.0.10-1) ...
Preparing to unpack .../nginx-common_1.26.0-1_all.deb ...
Unpacking nginx-common (1.26.0-1) over (1.24.0-2) ...
Preparing to unpack .../nginx_1.26.0-1_amd64.deb ...
Unpacking nginx (1.26.0-1) over (1.24.0-2+b1) ...
dpkg: libssl3:amd64: dependency problems, but removing anyway as you requested:
 wpasupplicant depends on libssl3 (>= 3.0.0).
 vboot-utils depends on libssl3 (>= 3.0.0).
 vboot-kernel-utils depends on libssl3 (>= 3.0.0).
 tnftp depends on libssl3 (>= 3.0.0).
 thc-pptp-bruter depends on libssl3 (>= 3.0.0).
 thc-ipv6 depends on libssl3 (>= 3.0.0).
 tcpdump depends on libssl3 (>= 3.0.0).
 systemd depends on libssl3 (>= 3.0.0).
 stunnel4 depends on libssl3 (>= 3.0.0).
 sslsplit depends on libssl3 (>= 3.0.0).
 ssldump depends on libssl3 (>= 3.0.0).
 socat depends on libssl3 (>= 3.0.0).
 snmp depends on libssl3 (>= 3.0.0).
 skipfish depends on libssl3 (>= 3.0.0).
 samdump2 depends on libssl3 (>= 3.0.0).
 rsync depends on libssl3 (>= 3.0.0).
 redis-tools depends on libssl3 (>= 3.0.0).
 python3-ntp depends on libssl3 (>= 3.0.0).
 python3-cryptography depends on libssl3 (>= 3.0.0).
 python3-aioquic depends on libssl3 (>= 3.0.0).
 proxytunnel depends on libssl3 (>= 3.0.0).
 ppp depends on libssl3 (>= 3.0.0).
 powershell depends on libssl3 | libssl1.1 | libssl1.0.2 | libssl1.0.0; however:
  Package libssl3:amd64 is to be removed.
  Package libssl1.1 is not installed.
  Package libssl1.0.2 is not installed.
  Package libssl1.0.0 is not installed.
 postgresql-client-16 depends on libssl3 (>= 3.0.0).
 postgresql-client-15 depends on libssl3 (>= 3.0.0).
 postgresql-16 depends on libssl3 (>= 3.0.0).
 postgresql-15 depends on libssl3 (>= 3.0.0).
 php8.2-common depends on libssl3 (>= 3.0.0).
 php8.2-cli depends on libssl3 (>= 3.0.0).
 perl-openssl-defaults:amd64 depends on libssl3 (>= 3.0.0).
 passing-the-hash depends on libssl3 (>= 3.0.0).
 ophcrack-cli depends on libssl3 (>= 3.0.0).
 ophcrack depends on libssl3 (>= 3.0.0).
 openvpn depends on libssl3 (>= 3.0.0).
 openssh-server depends on libssl3 (>= 3.0.10).
 openssh-client depends on libssl3 (>= 3.0.10).
 opensc-pkcs11:amd64 depends on libssl3 (>= 3.0.0).
 opensc depends on libssl3 (>= 3.0.0).
 openfortivpn depends on libssl3 (>= 3.0.0).
 ntpsec depends on libssl3 (>= 3.0.0).
 nmap depends on libssl3 (>= 3.0.0).
 network-manager-l2tp-gnome depends on libssl3 (>= 3.0.0).
 network-manager-l2tp depends on libssl3 (>= 3.0.0).
 ncrack depends on libssl3 (>= 3.0.0).
 mupdf-tools depends on libssl3 (>= 3.0.0).
 mtd-utils depends on libssl3 (>= 3.0.0).
 mosquitto depends on libssl3 (>= 3.0.0).
 medusa depends on libssl3 (>= 3.0.0).
 mariadb-server-core depends on libssl3 (>= 3.0.0).
 mariadb-server depends on libssl3 (>= 3.0.0).
 mariadb-client-core depends on libssl3 (>= 3.0.0).
 mariadb-client depends on libssl3 (>= 3.0.0).
 libzip4:amd64 depends on libssl3 (>= 3.0.0).
 libyara10:amd64 depends on libssl3 (>= 3.0.0).
 libxmlsec1-openssl:amd64 depends on libssl3 (>= 3.0.0).
 libwinpr2-2:amd64 depends on libssl3 (>= 3.0.0).
 libwebsockets19:amd64 depends on libssl3 (>= 3.0.0).
 libunbound8:amd64 depends on libssl3 (>= 3.0.0).
 libtss2-esys-3.0.2-0:amd64 depends on libssl3 (>= 3.0.0).
 libsystemd-shared:amd64 depends on libssl3 (>= 3.0.0).
 libstrongswan-standard-plugins depends on libssl3 (>= 3.0.0).
 libssh2-1:amd64 depends on libssl3 (>= 3.0.0).
 libssh-4:amd64 depends on libssl3 (>= 3.0.0).
 libsnmp40:amd64 depends on libssl3 (>= 3.0.0).
 libshout3:amd64 depends on libssl3 (>= 3.0.0).
 libserf-1-1:amd64 depends on libssl3 (>= 3.0.0).
 libsasl2-modules:amd64 depends on libssl3 (>= 3.0.0).
 libsasl2-2:amd64 depends on libssl3 (>= 3.0.0).
 libruby3.1:amd64 depends on libssl3 (>= 3.0.0).
 librados2 depends on libssl3 (>= 3.0.0).
 librabbitmq4:amd64 depends on libssl3 (>= 3.0.0).
 libqt6network6:amd64 depends on libssl3.
 libqt5network5:amd64 depends on libssl3.
 libpython3.11-minimal:amd64 depends on libssl3 (>= 3.0.0).
 libpython2.7-stdlib:amd64 depends on libssl3 (>= 3.0.0).
 libpq5:amd64 depends on libssl3 (>= 3.0.0).
 libpkcs11-helper1:amd64 depends on libssl3 (>= 3.0.0).
 libpipewire-0.3-modules:amd64 depends on libssl3 (>= 3.0.0).
 libpaho-mqtt1.3:amd64 depends on libssl3 (>= 3.0.0).
 libopusfile0:amd64 depends on libssl3 (>= 3.0.0).
 libnvme1 depends on libssl3 (>= 3.0.0).
 libnode108:amd64 depends on libssl3 (>= 3.0.0).
 libnet-ssleay-perl:amd64 depends on libssl3 (>= 3.0.0).
 libnet-dns-sec-perl depends on libssl3 (>= 3.0.0).
 libneon27:amd64 depends on libssl3 (>= 3.0.0).
 libmosquitto1:amd64 depends on libssl3 (>= 3.0.0).
 libmongoc-1.0-0 depends on libssl3 (>= 3.0.0).
 libmariadb3:amd64 depends on libssl3 (>= 3.0.0).
 libkrb5-3:amd64 depends on libssl3 (>= 3.0.0).
 libkmod2:amd64 depends on libssl3 (>= 3.0.0).
 libjim0.82:amd64 depends on libssl3 (>= 3.0.0).
 libimobiledevice6:amd64 depends on libssl3 (>= 3.0.0).
 libhdf5-103-1:amd64 depends on libssl3 (>= 3.0.0).
 libhashkit2:amd64 depends on libssl3 (>= 3.0.0).
 libglusterfs0:amd64 depends on libssl3 (>= 3.0.0).
 libgdal33 depends on libssl3 (>= 3.0.0).
 libfsverity0:amd64 depends on libssl3 (>= 3.0.0).
 libfreerdp2-2:amd64 depends on libssl3 (>= 3.0.0).
 libfido2-1:amd64 depends on libssl3 (>= 3.0.0).
 libevent-openssl-2.1-7:amd64 depends on libssl3 (>= 3.0.0).
 libeac3:amd64 depends on libssl3 (>= 3.0.0).
 libcurl4:amd64 depends on libssl3 (>= 3.0.0).
 libcryptsetup12:amd64 depends on libssl3 (>= 3.0.0).
 libcrypt-ssleay-perl depends on libssl3 (>= 3.0.0).
 libaprutil1:amd64 depends on libssl3 (>= 3.0.0).
 libapache2-mod-php8.2 depends on libssl3 (>= 3.0.0).
 libafflib0v5:amd64 depends on libssl3 (>= 3.0.0).
 kmod depends on libssl3 (>= 3.0.0).
 kismet-core depends on libssl3 (>= 3.0.0).
 john depends on libssl3 (>= 3.0.0).
 ike-scan depends on libssl3 (>= 3.0.0).
 hydra depends on libssl3 (>= 3.0.0).
 gstreamer1.0-plugins-good:amd64 depends on libssl3 (>= 3.0.0).
 gstreamer1.0-plugins-bad:amd64 depends on libssl3 (>= 3.0.0).
 gss-ntlmssp depends on libssl3 (>= 3.0.0).
 galera-4 depends on libssl3 (>= 3.0.0).
 ettercap-common depends on libssl3 (>= 3.0.0).
 dotnet-runtime-deps-6.0 depends on libssl1.0.0 | libssl1.0.2 | libssl1.1 | libssl3; however:
  Package libssl1.0.0 is not installed.
  Package libssl1.0.2 is not installed.
  Package libssl1.1 is not installed.
  Package libssl3:amd64 is to be removed.
 bind9-libs:amd64 depends on libssl3 (>= 3.0.0).
 axel depends on libssl3 (>= 3.0.0).
 apache2-utils depends on libssl3 (>= 3.0.0).
 apache2-bin depends on libssl3 (>= 3.0.0).

(Reading database ... 471418 files and directories currently installed.)
Removing libssl3:amd64 (3.0.10-1) ...
Selecting previously unselected package libssl3t64:amd64.
(Reading database ... 471406 files and directories currently installed.)
Preparing to unpack .../libssl3t64_3.2.1-3_amd64.deb ...
Unpacking libssl3t64:amd64 (3.2.1-3) ...
Setting up libssl3t64:amd64 (3.2.1-3) ...
Setting up libssl-dev:amd64 (3.2.1-3) ...
Setting up openssl (3.2.1-3) ...
Installing new version of config file /etc/ssl/openssl.cnf.original ...
Setting up nginx (1.26.0-1) ...
Warning: The unit file, source configuration file or drop-ins of nginx.service changed on disk. Run 'systemctl daemon-reload' to reload units.
Upgrading binary: nginx.
Setting up nginx-common (1.26.0-1) ...
Installing new version of config file /etc/nginx/mime.types ...
Installing new version of config file /etc/nginx/nginx.conf ...
Installing new version of config file /etc/ufw/applications.d/nginx ...
Processing triggers for man-db (2.11.2-3) ...
Processing triggers for kali-menu (2023.4.5) ...
Processing triggers for libc-bin (2.37-8) ...
Scanning processes...                                                                                                                                        
Scanning candidates...                                                                                                                                       
Scanning linux images...                                                                                                                                     

Running kernel seems to be up-to-date.

Restarting services...
 /etc/needrestart/restart.d/systemd-manager
 systemctl restart acunetix.service systemd-journald.service systemd-udevd.service udisks2.service virtualbox-guest-utils.service
Job for virtualbox-guest-utils.service failed because the control process exited with error code.
See "systemctl status virtualbox-guest-utils.service" and "journalctl -xeu virtualbox-guest-utils.service" for details.
Service restarts being deferred:
 systemctl restart NetworkManager.service
 systemctl restart systemd-logind.service

No containers need to be restarted.

User sessions running outdated binaries:
 kali @ session #2: sh[147906], xfce4-panel[1933], xfce4-session[1780]
 kali @ user manager service: systemd[1732]

No VM guests are running outdated hypervisor (qemu) binaries on this host.
'/opt/asf/tools/nginx/sites-enabled/asf' -> '/etc/nginx/sites-enabled/asf'
ln: failed to create symbolic link '/etc/nginx/sites-available/asf': File exists
ln: failed to create symbolic link '/bin/startasf.sh': File exists
A.S.F. Running on: http://127.0.0.1:2021
